frontend/src/components/worksheets/items/TableItem/TableItem.js

```diff
@@ -159,9 +159,16 @@ class TableItem extends React.Component<{
             if (rowItems[0][x] && rowItems[0][x]['path']) columnWithHyperlinks.push(x);
         });
         var bodyRowsHtml = rowItems.map((rowItem, rowIndex) => {
-            var rowRef = 'row' + rowIndex;
```

Comment:
Ignore this, merging for dev deployment

Reply:
You mean ignore all changes in this file?

Will these changes be gone in the diff before we merge?

Reply:
These changes would be introduced by #1754